app/assets/javascripts/models/knockout_extensions.js.coffee

```diff
@@ -207,3 +207,18 @@ do (ko, $=jQuery) ->
     ko.bindingHandlers.text.update element, ->
       truncatedText
     return
+
+  ko.bindingHandlers.descriptionLinks =
+    init: (element, valueAccessor, allBindings, viewModel, bindingContext) ->
+      description = $($.parseHTML(bindingContext.$data.description))
+      for el, i in description
+        if el.nodeName == 'A' && el.host != window.location.host
+          $(el).attr('target', '_blank')
+          $(el).attr('class', 'ext')
+          description[i] = el.outerHTML
+        else if el.nodeName == 'A' && el.host == window.location.host
+          description[i] = $(el.outerHTML).text()
+        else
+          description[i] = $(el).text()
+
+      ko.applyBindingsToNode(element, { 'html': (el for el in description).join('')}, bindingContext)
```

Comment:
That would have been a good option, but `.join('')` doesn't work because `description` isn't really an Array - it's a DOM Nodelist so `.join()` isn't a valid function.

Reply:
👍